src/locales/en.json

```diff
@@ -18,7 +18,9 @@
     "platformNotSupported": "Platform not supported for this request",
     "serverError": "An error occurred while processing your request",
     "cacheCheckError": "An error occurred while processing your request",
-    "print": "Failed to print, please try again."
+    "print": "Failed to print, please try again.",
+    "shareFiles_one": "We couldn't share your file, please try again.",
+    "shareFiles_other": "We couldn't share your files, please try again."
```

Comment:
In other cozy-apps we have a "plural" form that is declared in the same translation key. Like in this example: 
- locale: https://github.com/cozy/cozy-ui/blob/5900e4fd30ab044fd7ea717f672095337abebbd1/react/UploadQueue/locales/en.json#L4
- code usage: https://github.com/cozy/cozy-ui/blob/5900e4fd30ab044fd7ea717f672095337abebbd1/react/UploadQueue/index.jsx#L287-L290

Do you think we can use the same form here?

Reply:
I can look into it, would be better to have same syntax indeed

- [ ] todo

Reply:
using https://www.i18next.com/translation-function/plurals#interval-plurals

Reply:
Seems weird that we still need the `_other` key but ok.

Reply:
@Ldoppea I think it's to handle wrong count like non-numbers, or zero. Not sure how it's handled on cozy-ui I can take a look